gonka/chain.json

```diff
@@ -0,0 +1,220 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "gonka",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Gonka",
+  "chain_type": "cosmos",
+  "chain_id": "gonka-mainnet",
+  "bech32_prefix": "gonka",
+  "daemon_name": "inferenced",
+  "node_home": "$HOME/.inference",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 1200,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ngonka",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0,
+        "high_gas_price": 0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ngonka"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/gonka-ai/gonka",
+    "language": {
+      "type": "go" 
+    },
+    "genesis": {
+      "genesis_url": "https://github.com/gonka-ai/gonka/blob/main/genesis/genesis.json"
+    }
+  },
+  "images": [
+    {
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/gonka/images/gonka_logo.svg",
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/gonka/images/gonka_logo.png",
+      "theme": {
+        "circle": true,
+        "primary_color_hex": "#000000"
```

Comment:
Color data is not longer accepted at the Chain Registry. This is making schema validation fail